### PR TITLE
fix(Login): show promotions depend of the page parameter

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const forcedServices =
   process.env.NODE_ENV === 'development' || process.env.REACT_APP_IS_DEMO_ENABLED
     ? {
         dopplerLegacyClient: new HardcodedDopplerLegacyClient(),
-        dopplerSitesClient: new HardcodedDopplerSitesClient(),
+        //dopplerSitesClient: new HardcodedDopplerSitesClient(),
```
Comment (cbernat) [inline src/index.js]: Here you need to remove the comment for it to work in demo and dev right?

Comment (andresmoschini) [inline src/index.js]: is it fine?

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -261,7 +266,7 @@ const Login = ({ location, dependencies: { dopplerLegacyClient, sessionManager,
           </p>
         </footer>
       </article>
-      <Promotions type="login" />
+      <Promotions type="login" page={extractPage(location)} />
```
Comment (andresmoschini) [inline src/components/Login/Login.js]: ❤️ 🎉

**Comment by cbernat** [comment]
Nice!

File: src/index.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/index.js]: i correct, thanks

Comment (fcoronelmakingsense) [reply src/index.js]: there is wrong! thanks for the advice

**Pull request merged**